aws/ecr/outputs.tf

```diff
@@ -77,3 +33,12 @@ output "ecr_repository_url_api" {
   description = "URL of the Forms API's ECR"
   value       = aws_ecr_repository.api.repository_url
 }
+
+output "ecr_repository_lambda_urls" {
+  description = "URL Map of the Lambda's ECR"
+  value = var.env == "development" ? {
+    for lambda_name, ecr_repository in aws_ecr_repository.lambda : lambda_name => replace(ecr_repository.repository_url, var.account_id, var.staging_account_id)
+    } : {
+    for lambda_name, ecr_repository in aws_ecr_repository.lambda : lambda_name => tostring(ecr_repository.repository_url)
```

Comment:
Just because I'm curious, what was happening when you didn't have this wrapped in `tostring()`?  I would have expected that `repository_url` attribute to behave as a string by default.

Reply:
So did I but instead it was throwing an error that the `ecr_repository` object did not have a `repository_url` property.  Logging out the `ecr_repository` clearly showed it did.  So as a test I found the most benign terraform function (`tostrign`) to see if it behaved any differently.  Sure enough it was now correctly parsing the string.

Reply:
Here's the run with the error: https://github.com/cds-snc/forms-terraform/actions/runs/13551795444/job/37876882524

Reply:
Very weird but glad the `tostring` fixed it.